Dockerfile

```
@@ -0,0 +1,19 @@
+FROM          python:3.7-alpine
+
+RUN           mkdir /app
+COPY          requirements.txt /app
+
+RUN           apk --no-cache add --virtual build-dependencies \
+                build-base \
+                py-mysqldb \
+                gcc \
+                libc-dev \
+                libffi-dev \
+                mariadb-dev \
+                && pip install -qq -r /app/requirements.txt \
+                && rm -rf .cache/pip \
+                && apk del build-dependencies
+
+COPY          app.py /app
```

This can be reduced in line 4. Remove line 3 (as ADD creates it).

```suggestion
ADD          requirements.txt app.py /app
```